annotations.py

```diff
@@ -190,11 +192,18 @@ def to_HTML(self, header=''):
                 except:
                     if agroup.highlightcolor is None:
                         msg = "No highlight color specified, using Default"
+                        dt_bgcolor = COLOR_MAP['Default']['bg']
+                        dt_fgcolor = COLOR_MAP['Default']['fg']
+                    elif htmlColorPattern.match(agroup.highlightcolor):
+                        msg = "Found valid RGB(A) HTML color '%s', using" % agroup.highlightcolor
+                        dt_bgcolor = agroup.highlightcolor
+                        dt_fgcolor = COLOR_MAP['Default']['fg']
```

Comment:
Not all background colors look well with the black foreground color. Therefore it's better to add support for specifying a custom foreground color (optional setting, that defaults to `COLOR_MAP['Default']['fg']` as now). 

---

Changes in this file look like a nice feature and should go into a standalone PR with an explanation of how to use that feature. Probably sample annotation importer file also should be updated with an examples.

Reply:
In the case of moon, there's no foreground color so where am I supposed to set this, a plugin-wide setting?

Reply:
I suppose it could go under "Modify appearance" to set a default highlight foreground/background? I can put together a separate PR for this.

Reply:
#24

Reply:
#24 reviewed.